config/commands.js

```diff
@@ -674,35 +690,54 @@ var commands = exports.commands = {
 					}
 					break;
 
-				case 'moves':
+				case 'compileLearnsets':
```

Comment:
Because combining the prevo Learnsets is the slowest part of the command I wanted to run this as late in the search as possible to reduce the number of pokemon who's learnsets had to be created to a minimum. However, adding it as a case feels a bit inappropriate so I'm wondering if the switch statement should be removed and replaced with a series of `if (searches[<search parameter>])`